### DISCUSSION TITLE
Website refresh and GitHub Discussions

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#21880] [Quarkus Blog/Website]
# Website refresh and GitHub Discussions

Quarkus: Supersonic Subatomic Java

http://quarkus.io/blog/website-and-discussions/

**Comment by emmanuelbernard** [comment]
I'm not super clear on when to use the Community category vs when to use Q&A. I wonder how we could clarify it further.
The other thing I felt is that Q&A might be the most important category but it's deep down the list.
That's what came to mind when I read the post.

**Reply by maxandersen** [reply]
I wish GitHub discussions would let us reorder the category order but haven't found a way to do that. For now they order alphabetically. 

About when to use community or Q&A I'm not sure how to make it clearer. For me if question about usage of quarkus the use q&a. For anything else use community. Ie. If you have an idea or some proposal for something.

**Comment by emmanuelbernard** [comment]
Another question that popped up is that I did not see a way to filter only Announcements discussions. The custom filter in GitHub allws for all discussions or none. Is there a more fine grained menu I'm not seeing?

**Reply by maxandersen** [reply]
To see only Announcements go to https://github.com/quarkusio/quarkus/discussions/categories/announcements

For more control its the labels - I've considered that if it gets too noisy we could auto add a matching label, like "cat/announcements"  so can use that.
